### DISCUSSION TITLE
What are the supplied jumpers for?

### DISCUSSION THREAD

**Discussion by dJOS1475** [thehighnibble/thehighnibble.github.io#36] [Q&A] [answered]
I’m prolly missing something in the documentation, but I have a bunch of unused jumpers from the kit and no idea what they are for.

**Comment by dmcnaugh** [comment, marked as answer]
If you are going to use either of the two RS-232 DE-9 connectors on the back of the IMSAI8080esp to connect a serial device you need jumpers - 4 jumpers for each port. See https://thehighnibble.com/imsai8080/configure/#serial-uart-over-rs232

**Reply by dJOS1475** [reply]
Cheers, makes sense.
